daemon/config/config.go

```diff
@@ -56,6 +56,11 @@ const (
 	DefaultContainersNamespace = "moby"
 	// DefaultPluginNamespace is the name of the default containerd namespace used for plugins.
 	DefaultPluginNamespace = "plugins.moby"
+	// DefaultAPIVersion is the highest REST API version supported by the daemon.
+	//
+	// This version may be lower than the [api.DefaultVersion], which is the default
+	// (and highest supported) version of the api library module used.
+	DefaultAPIVersion = "1.52"
```

Comment:
Possibly these should be moved to the `Server` package and/or injected to it, but we can move later if we see fit.

Reply:
If we'd go for `go generate`, we'd have to look where to put that code (i.e., we don't want someone using the `client` module to run `go generate` and it picking up the version from the `api` module 🤔

Reply:
IMO, `go generate` would just be a part of the "bump API version" action on the maintainer side.